ui/src/main/webapp/lib/admin-wizard/inputs.js

```diff
@@ -113,61 +123,67 @@ const Port = ({ value = 0, label = 'Port', ...rest }) => (
-      <SelectField
-        fullWidth
-        errorText={error}
-        value={i}
-        onChange={(e, i) => onEdit(options[i])}
-        floatingLabelText={label}
-        {...rest}>
-        {options.map((d, i) => {
-          if (typeof d === 'string') {
-            return <MenuItem key={i} value={i} primaryText={d} />
-          } else if (typeof d === 'object') {
-            return <MenuItem key={i} value={i} primaryText={d.name} />
-          } else {
-            return null
-          }
-        })}
-      </SelectField>
-      {(tooltip)
-        ? (<IconButton style={{ position: 'absolute', right: '10px', bottom: '0px' }}
-          iconStyle={{ width: '20px', height: '20px' }}
-          tooltip={tooltip}
-          touch
-          tooltipPosition='top-left'>
-          <InfoIcon />
-        </IconButton>)
-        : null
-      }
-    </Flexbox>
-  )
+  if (visible) {
+    return (
+      <Flexbox flexDirection='row' style={{position: 'relative'}}>
+        <SelectField
+          fullWidth
+          errorText={error}
+          value={i}
+          onChange={(e, i) => onEdit(options[i])}
+          floatingLabelText={label}
+          {...rest}>
+          {options.map((d, i) => {
+            if (typeof d === 'string') {
+              return <MenuItem key={i} value={i} primaryText={d} />
+            } else if (typeof d === 'object') {
+              return <MenuItem key={i} value={i} primaryText={d.name} />
+            } else {
+              return null
+            }
+          })}
+        </SelectField>
+        {(tooltip)
+          ? (<IconButton style={{position: 'absolute', right: '10px', bottom: '0px'}}
+            iconStyle={{width: '20px', height: '20px'}}
+            tooltip={tooltip}
+            touch
+            tooltipPosition='top-left'>
+            <InfoIcon />
+          </IconButton>)
+          : null
+        }
+      </Flexbox>
+    )
+  }
+  return null
 }
 
 const Select = connect(mapStateToProps, mapDispatchToProps)(SelectView)
 
-const RadioSelectionView = ({ value, disabled, options = [], onEdit, ...rest }) => {
+const RadioSelectionView = ({ value, disabled, options = [], onEdit, visible = true, ...rest }) => {
   for (let key in rest) {
     if (rest[key] === undefined) {
       delete rest[key]
     }
   }
-  return (
-    <RadioButtonGroup valueSelected={value} onChange={(e, value) => onEdit(value)} {...rest}>
-      {options.map((item, i) =>
-        <RadioButton style={{ fontSize: '16px', padding: '3px' }} key={i} value={item.value} label={item.label} disabled={disabled} />)}
-    </RadioButtonGroup>
-  )
+  if (visible) {
+    return (
+      <RadioButtonGroup valueSelected={value} onChange={(e, value) => onEdit(value)} {...rest}>
+        {options.map((item, i) =>
+          <RadioButton style={{fontSize: '16px', padding: '3px'}} key={i} value={item.value}
+            label={item.label} disabled={disabled} />)}
+      </RadioButtonGroup>
+    )
+  }
```

Comment:
Yes! Good catch, should have a `return null` at the end - jsx elements aren't supposed to return undefined.